broker/setup_broker/lsst/setup_broker.sh

```diff
@@ -81,7 +83,10 @@ manage_resources() {
         bq --location="${region}" mk --dataset "${bq_dataset}"
 
         (cd templates && bq mk --table "${PROJECT_ID}:${bq_dataset}.${alerts_table}" "bq_${survey}_${alerts_table}_schema.json") || exit 5
+        (cd templates && bq mk --table "${PROJECT_ID}:${bq_dataset}.${supernnova_table}" "bq_${survey}_${supernnova_table}_schema.json") || exit 5
+        (cd templates && bq mk --table "${PROJECT_ID}:${bq_dataset}.${classifications_table}" "bq_${survey}_${classifications_table}_schema.json") || exit 5
```

Comment:
Let's revisit this table structure. I think it would be better for both us and users to have one wide table rather than separating things out into separate tables. 

I wonder if we can come up with a table structure and an alert structure that match so that we don't have to publish a separate topic for the bigquery subscriptions. I'm thinking of something that probably has a top-level field for each module, each with a set of fields nested underneath for its own data.